trimmer_decorator.rb

```diff
@@ -0,0 +1,8 @@
+require_relative 'decorator'
+
+class TrimmerDecorator < Decorator
+  def correct_name
+    trimmed_name = @nameable.correct_name[0...10]
+    trimmed_name.length < @nameable.correct_name.length ? "#{trimmed_name}..." : trimmed_name
```

Comment:
- [ ] [OPTIONAL] I suggest you stick to the requirements, here you shouldn't add `...` after the trimmed name
```suggestion
    trimmed_name.length < @nameable.correct_name.length ? "#{trimmed_name}" : trimmed_name
```